arrow/src/main/java/org/apache/iceberg/arrow/vectorized/parquet/VectorizedParquetDefinitionLevelReader.java

```diff
@@ -447,6 +448,42 @@ protected void nextDictEncodedVal(
     }
   }
 
+  class TimestampInt96Reader extends BaseReader {
+    @Override
+    protected void nextVal(
+        FieldVector vector,
+        int idx,
+        ValuesAsBytesReader valuesReader,
+        int typeWidth,
+        byte[] byteArray) {
+      ByteBuffer buffer = valuesReader.getBuffer(12);
```

Comment:
maybe add a comment what the 12 means here (8 bytes = time of day in nanos / 4 bytes = julian day)

Reply:
done